core/src/actions/destinations.ts

```diff
@@ -176,6 +177,20 @@ export class DestinationEdit extends AuthenticatedAction {
 
     await destination.update(params);
 
+    if (
+      params.trackedGroupId &&
+      params.trackedGroupId !== "_none" &&
```

Comment:
This `'_none'` thing is kind of janky, but the alternative would be to have another param like `unTrackedGroupId`, and the whole point is to remove this logic from the client... so I'm 👍 with it!

Reply:
Yeeah, those were my thoughts as well